doc/release-notes.md

```diff
@@ -102,6 +102,17 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
   option `-deprecatedrpc=banscore` is used. The `banscore` field will be fully
   removed in the next major release. (#19469)
 
+- The `getpeerinfo` RPC now returns a `connection_type` field. This indicates
+  the type of connection established with the peer. It will return one of six
+  options. For more information, see the `getpeerinfo` help documentation.
```

Comment:
While these are release notes, I still think it's worth it to at least list those 6 options to give a hint at what they are, even if describing each may be too verbose.

Reply:
In general, I think we try to avoid duplicating documentation in this way. Release notes don't need to include all the details, and pointing where to go for the source of truth is appropriate here.

Reply:
Only reading the release notes you are left wanting, I suspect; explicitly saying there's six of them makes you start to wonder "so many? I can think of maybe 3 at most... curious!".

Half-joke aside, simply "one of six options: a, b, c, d, e, or f" would be useful, but no strong opinion on the matter.

Reply:
Agree with @jnewbery. Release notes should describe the change, briefly but clearly, they do not need to provide documentation on how to use something. I think this description is okay.

Reply:
ok, seems like the dominant reviewer preference is to leave as is? so, resolving this conversation